src/locale/mr.js

```diff
@@ -52,19 +89,19 @@ export default moment.defineLocale('mr', {
         sameElse : 'L'
     },
     relativeTime : {
-        future : '%s नंतर',
-        past : '%s पूर्वी',
-        s : 'सेकंद',
-        m: 'एक मिनिट',
-        mm: '%d मिनिटे',
-        h : 'एक तास',
-        hh : '%d तास',
-        d : 'एक दिवस',
-        dd : '%d दिवस',
-        M : 'एक महिना',
-        MM : '%d महिने',
-        y : 'एक वर्ष',
-        yy : '%d वर्षे'
+        future: '%sमध्ये',
```

Comment:
I did think of that and honestly, I think when it is a plural, it should always have अनुस्वार. Since अनुस्वार is what pluralizes it. But Marathi is super-dynamic language, you can find examples of both ways.